swarm/network/stream/common_test.go

```diff
@@ -319,8 +360,17 @@ func (c *testExternalClient) NeedData(hash []byte) func() {
 	}
 }
 
-func (c *testExternalClient) BatchDone(Stream, uint64, []byte, []byte) func() (*TakeoverProof, error) {
-	return nil
+func (c *testExternalClient) BatchDone(s Stream, from uint64, hashes []byte, root []byte) func() (*TakeoverProof, error) {
+	return func() (*TakeoverProof, error) {
```

Comment:
what is this used for?

Reply:
This can be removed, it was my try to debug and was probably included in a patch that I sent to Fabio. It is not needed.